src/ResourceManager/AnalysisServices/Commands.AnalysisServices.Test/ScenarioTests/AsTests.ps1

```diff
@@ -528,42 +544,130 @@ function Test-AnalysisServicesServerRestart
 	}
 }
 
+<#
+.SYNOPSIS
+Tests associate/dissociate gateway for server
+In order to run this test successfully, Following environment variables need to be set.
+A pre-configured onpremise gateway that is configured in the same location for testing default is East US 2.
+The testing user should have the permission to the gateway.
+ex. SET ASAZURE_TEST_ADMUSERS=asengsys@microsoft.com
+    SET ASAZURE_TESTUSER_PWD=somepassword
+	SET ASAZURE_TEST_ROLLOUT=stable.asazure.windows.net
+	SET ASAZURE_TEST_GATEWAY_NAME=azuresdktest
+#>
+function Test-AnalysisServicesServerGatewayAssociation
+{
+	try
+	{  
+		# Creating server
+		$location = "West US"
```

Comment:
Please use Get-Location to ensure this works across different azure domains: https://github.com/Azure/azure-powershell/blob/preview/src/ResourceManager/Common/Commands.ScenarioTests.ResourceManager.Common/Common.ps1#L583.  Please be sure to specify all three parameters.

Reply:
this is not get resource location or provider location, this is the target location we would like to create Azure resource.